DSCResources/MSFT_xDSCWebService/MSFT_xDSCWebService.psm1

```diff
@@ -161,8 +165,11 @@ function Set-TargetResource
     if ($IsBlue)
     {
         Write-Verbose "Set values into the web.config that define the repository for BLUE OS"
-        PSWSIISEndpoint\Set-AppSettingsInWebconfig -path $PhysicalPath -key "dbprovider" -value $eseprovider
-        PSWSIISEndpoint\Set-AppSettingsInWebconfig -path $PhysicalPath -key "dbconnectionstr"-value $esedatabase
+        #PSWSIISEndpoint\Set-AppSettingsInWebconfig -path $PhysicalPath -key "dbprovider" -value $eseprovider
```

Comment:
I tested some more. When the Pull Server is installed with WMF4 not yet on the system, the edb file gets created. When WMF5 is already installed, the edb file is however not created / placed and a pull from a client results in an ASP error of file not found. It could therefore be a bug in WMF5 and in that case, only the culture fix should remain.